cfgov/unprocessed/css/molecules/image-text-50-50.less

```diff
@@ -82,7 +82,6 @@
 */
 
 .m-image-text-50-50 {
-    padding-bottom: unit( @grid_gutter-width * 2 / @base-font-size-px, em );
```

Comment:
Drat. The CMS does let you add six to one group. If we define it at the group level we'll need to fix that, but it's easier to do what Jimmy's suggesting.